{"name": "test_login_negative", "status": "broken", "statusDetails": {"message": "AttributeError: module 'lib2to3.pgen2.driver' has no attribute 'find_element'", "trace": "Setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"1ee6dcd5f2708086372439d9dc883c3a\")>\n\n    def test_login_negative(Setup):\n        login = LoginPage(driver)\n        time.sleep(5)\n>       login.login_to_vwo(email=\"vwo@app.vwo.com\", pwd=\"admin\")\n\ntest_vwo_login.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pageObjects\\loginPage.py:42: in login_to_vwo\n    self.get_email_address().send_keys(email)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <tests.pageObjects.loginPage.LoginPage object at 0x000001DA2D8EECD0>\n\n    def get_email_address(self):\n>       return self.driver.find_element(*LoginPage.email_address)\nE       AttributeError: module 'lib2to3.pgen2.driver' has no attribute 'find_element'\n\n..\\pageObjects\\loginPage.py:22: AttributeError"}, "start": 1728028169254, "stop": 1728028174266, "uuid": "8a92c04b-f716-4b55-bb9f-b5fae29bab6c", "historyId": "8be6628af41c999eb267bfd17245a1df", "testCaseId": "8be6628af41c999eb267bfd17245a1df", "fullName": "tests.vwoLoginTests.test_vwo_login#test_login_negative", "labels": [{"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwo_login"}, {"name": "host", "value": "DESKTOP-O5T1HPJ"}, {"name": "thread", "value": "12876-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.test_vwo_login"}]}